source/scripts/Background/helpers/assetsBalanceMonitor.ts

```diff
@@ -123,12 +123,20 @@ export class AssetsBalanceMonitor {
   async refreshL0balances(l0assets: IAssetInfoState[], dagAddress: string) {
     let l0balances = {};
     for (const l0asset of l0assets) {
-      const balance = (await dag4.network.blockExplorerV2Api.getCurrencyAddressBalance(l0asset.address, dagAddress) as any )?.data?.balance ?? 0;
-      const balanceNumber = new BigNumber(balance).multipliedBy(DAG_DECIMAL_FACTOR).toNumber();
+      const balance =
+        (
+          (await dag4.network.blockExplorerV2Api.getCurrencyAddressBalance(
+            l0asset.address,
+            dagAddress
+          )) as any
+        )?.data?.balance ?? 0;
+      const balanceNumber = new BigNumber(balance)
+        .multipliedBy(DAG_DECIMAL_FACTOR)
+        .toNumber();
```

Comment:
I think this snippet can be abstracted into an util rather than being written each time, something like `getAddressBlockExplorerBalance(...)`.